flink-core/src/main/java/org/apache/flink/configuration/NettyShuffleEnvironmentOptions.java

```diff
@@ -209,6 +210,11 @@
 			.withDeprecatedKeys("taskmanager.net.request-backoff.max")
 			.withDescription("Maximum backoff in milliseconds for partition requests of input channels.");
 
+	@Documentation.ExcludeFromDocumentation("dev use only; likely temporary")
+	public static final ConfigOption<Boolean> RELEASE_EXTERNALLY_MANAGED_PARTITIONS_ON_CONSUMPTION =
+			key("taskmanager.network.partition.release-externally-managed-on-consumption")
```

Comment:
key("taskmanager.network.release-externally-managed-partitions-on-consumption") to consistent with var naming.

Reply:
We don't have to be consistent with the variable name. The `partition` is already contained in the key prefix.